### ISSUE TITLE
Have I correctly used list mode?

### ISSUE THREAD

**Issue by calebkoy** [jpauwels/MusOOEvaluator#5]
This one should (hopefully) be a short one! I experimented with using list mode today and want to check that I've correctly understood and executed the instructions in [README.md](https://github.com/jpauwels/MusOOEvaluator).

My validation set contains two songs whose ground truth files are [anjing.txt](https://github.com/jpauwels/MusOOEvaluator/files/1791540/anjing.txt) and [bandaotiehe.txt](https://github.com/jpauwels/MusOOEvaluator/files/1791541/bandaotiehe.txt). The test files are [anjing.txt](https://github.com/jpauwels/MusOOEvaluator/files/1791544/anjing.txt) and [bandaotiehe.txt](https://github.com/jpauwels/MusOOEvaluator/files/1791545/bandaotiehe.txt) (as before, I use a neural network to predict the chord labels). From the instructions in the README, it seems as though a reference file and its corresponding test file needs to have the same name. Is that right? The list file I used is [list_file.txt](https://github.com/jpauwels/MusOOEvaluator/files/1791551/list_file.txt). 

I placed my reference files in `C:\Users\caleb\Desktop\CNNs\evaluation\reference_files` and my test files in `C:\Users\caleb\Desktop\CNNs\evaluation\test_files`. 

Then I ran `MusOOEvaluator.exe --list list_file.txt --refdir C:\Users\caleb\Desktop\CNNs\evaluation\reference_files --testdir C:\Users\caleb\Desktop\CNNs\evaluation\test_files --refext .txt --testext .txt --chords MirexSeventhsBass --output output_test.txt` And I got this output file: 
[output_test.txt](https://github.com/jpauwels/MusOOEvaluator/files/1791554/output_test.txt). The performance is not great, but I didn't expect much from this toy example. :D I just wanted to test that everything was working. 

Have I used list mode correctly?

Thank you in advance.

**Comment by jpauwels** [comment]
Entirely correct!

Just FYI, the reference and test file need to start with the same string (the part you put in `list_file.txt`), but you can append different suffixes, e.g. `anjing-ref.txt` and `anjing-test.txt` and then call the programme with `--refext -ref.txt --testext -test.txt`.

**Comment by calebkoy** [comment]
Great! Thank you!
